src/ontology/imports/cl_import.owl

```diff
@@ -23,13 +23,13 @@ Annotation(<http://purl.org/dc/terms/contributor> <https://orcid.org/0000-0002-9
 Annotation(<http://purl.org/dc/terms/contributor> <https://orcid.org/0000-0003-1980-3228>)
 Annotation(<http://purl.org/dc/terms/license> <http://creativecommons.org/licenses/by/4.0/>)
 Annotation(rdfs:comment "See PMID:15693950, PMID:12799354, PMID:20123131, PMID:21208450; Contact Alexander Diehl, addiehl@buffalo.edu, University at Buffalo.")
-Annotation(owl:versionInfo "2023-02-19")
+Annotation(owl:versionInfo "2023-05-22")
+Annotation(owl:versionInfo "2023-06-21")
```

Comment:
how does this have two version annotations?

Reply:
Hmm, the new pipeline is getting the CL ontology version and the CL import version.

Reply:
ODK issue: do not inject `owl:versionInfo` into imports?

Reply:
it seems that running the release again fixed this. (?)